tests/test_odl.py

```diff
@@ -1929,3 +1931,161 @@ def canonicalize_author_name(self, identifier, working_display_name):
+        :type collection: core.model.collection.Collection
+
+        :param http_get: Use this method to make an HTTP GET request.
+            This can be replaced with a stub method for testing purposes.
+        :type http_get: Callable
+
+        :return: ODLImporter object
+        :rtype: ODLImporter
+        """
+        importer = ODLImporter(
+            self._db,
+            collection=collection,
+            http_get=http_get,
+        )
+
+        return importer
+
+    def _get_test_feed_with_single_odl_license(self, expires):
+        """Get the feed with a single ODL license with the specific expiration date.
+
+        :param expires: Expiration date of the ODL license
+        :type expires: datetime.datetime
+
+        :return: Test ODL feed with a single ODL license with the specific expiration date
+        :rtype: str
+        """
+        feed = self.get_data(self.ODL_FEED_FILENAME_WITH_SINGLE_ODL_LICENSE)
+        feed = feed.replace(self.ODL_LICENSE_EXPIRATION_TIME_PLACEHOLDER, expires.isoformat())
+
+        return feed
+
+    def _import_test_feed_with_single_odl_license(self, expires):
+        """Import the test ODL feed with a single ODL license with the specific expiration date.
+
+        :param expires: Expiration date of the ODL license
+        :type expires: datetime.datetime
+
+        :return: 3-tuple containing imported editions, license pools and works
+        :rtype: Tuple[List[Edition], List[LicensePool], List[Work]]
+        """
+        feed = self._get_test_feed_with_single_odl_license(expires)
+        data_source = DataSource.lookup(self._db, "Feedbooks", autocreate=True)
+        collection = MockODLAPI.mock_collection(self._db, protocol=self.ODL_PROTOCOL)
+        collection.external_integration.set_setting(
+            Collection.DATA_SOURCE_NAME_SETTING,
+            data_source.name
+        )
+        license_status = {
+            "checkouts": {
+                "available": 1
+            }
+        }
+        license_status_response = MagicMock(return_value=(200, {}, json.dumps(license_status)))
+        importer = self._create_importer(collection, license_status_response)
+
+        imported_editions, imported_pools, imported_works, _ = (
+            importer.import_from_feed(feed)
+        )
+
+        return imported_editions, imported_pools, imported_works
+
+    @freeze_time("2021-01-01T00:00:00+00:00")
+    def test_odl_importer_skips_expired_licenses(self):
+        """Ensure ODLImporter skips expired licenses
+            and does not count them in the total number of available licenses."""
+        # 1.1. Import the test feed with an expired ODL license.
+        # The license expires 2021-01-01T00:01:00+01:00 that equals to 2010-01-01T00:00:00+00:00, the current time.
+        # It means the license had already expired at the time of the import.
+        license_expiration_date = datetime.datetime(2021, 1, 1, 1, 0, 0, tzinfo=tzoffset(None, 3600))
```

Comment:
The constant 3600 appears a couple of times. Might be helpful to make it a class constant like `SECONDS_PER_HOUR` or similar.

Reply:
I created `SECONDS_PER_HOUR` constant as requested, thank you for noticing this.